l2tdevtools/dependency_writers/gift_ppa.py

```diff
@@ -7,11 +7,11 @@
 
 from l2tdevtools.dependency_writers import interface
 
-
 class GIFTPPAInstallScriptWriter(interface.DependencyFileWriter):
-  """GIFT PPA installation script file writer."""
+  """GIFT PPA installation script file writer for Python 2."""
```

Comment:
I assume `for Python 2` needs to be removed here since there is a GIFTPPAInstallScriptPY2Writer ?

Maybe "Shared functionality for the GIFT PPA installation script file writer."""

Reply:
Done